testsuite/python/pair_criteria.py

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright (C) 2013,2014,2015,2016 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Tests particle property setters/getters
+from __future__ import print_function
+import unittest as ut
+import espressomd
+import numpy as np
+from espressomd.interactions import FeneBond
+from espressomd.pair_criteria import *
+
+
+class PairCriteria(ut.TestCase):
+    """Tests interface and implmentation of pair criteria"""   
+
+    es = espressomd.System()
+
+    f1 = FeneBond(k=1, d_r_max=5)
+    es.bonded_inter.add(f1)
+    f2 = FeneBond(k=1, d_r_max=5)
+    es.bonded_inter.add(f2)
+    es.part.add(id=0,pos=(0,0,0))
+    es.part.add(id=1,pos=(0.91,0,0))
+    p1=es.part[0]
+    p2=es.part[1]
+    # Setup purely repulsive lj
+    es.non_bonded_inter[0,0].lennard_jones.set_params(sigma=0.11,epsilon=1,cutoff=2**(1./6.)*0.11,shift="auto")
+    epsilon=1E-8
+
+    def test_distance_crit(self):
+        dc=DistanceCriterion(cut_off=0.1)
+        # Interface
+        self.assertTrue(dc.get_params().keys()==["cut_off",])
+        self.assertTrue(abs(dc.get_params()["cut_off"]-0.1)<self.epsilon)
```

Comment:
Please use
```python
self.assertAlmostEqual(a, b, places=7)
```
instead of
```python
self.assertTrue(abs(a-b)<self.epsilon)
```
Applies to all other comparisons with `epsilon` as well.